gooddata-java/src/test/java/com/gooddata/sdk/service/hierarchicalconfig/HierarchicalConfigServiceAT.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2004-2021, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.service.hierarchicalconfig;
+
+import com.gooddata.sdk.common.GoodDataException;
+import com.gooddata.sdk.model.hierarchicalconfig.SettingItem;
+import com.gooddata.sdk.model.hierarchicalconfig.Settings;
+import com.gooddata.sdk.model.hierarchicalconfig.SourceType;
+import com.gooddata.sdk.service.AbstractGoodDataAT;
+import org.testng.annotations.Test;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.hamcrest.Matchers.is;
+import static org.testng.Assert.fail;
+
+public class HierarchicalConfigServiceAT extends AbstractGoodDataAT {
+
+    private static final String FIRST_SETTING_ITEM = "flag1";
+    private static final String SECOND_SETTING_ITEM = "flag2";
+
+    @Test(groups = "projectSettings", dependsOnGroups = "project")
+    public void createProjectSetting() {
```

Comment:
Please update test methods to latest naming

Reply:
Changed